doc/upload-block-pv.md

```diff
@@ -0,0 +1,115 @@
+# How to upload an image to a block raw PV User Guide
+The purpose of this document is to show how to uploadan image from a URL (http/s3) without authentication.
+
+## Prerequisites
+- You have a Kubernetes cluster up and running with CDI installed and at least one PersistentVolume is available.
+- Feature-Gate 'BlockVolume' is enabled.
+
+## Upload an image to a Local raw block PV
+In case you do not upload an image to a local PV, please skip to the next section (Upload an image with DataVolume manifest).
+There are a few steps to follow, to be able to upload an image to a local raw block PV.
+
+First, create a local PV with the volumeMode field set to 'Block':
+
+```bash
```

Comment:
```yaml

Reply:
yaml tag instead of bash

Reply:
hmmm..... This is what we are using in all other .md's.
What's the difference?

Reply:
The highlighting of the text block. Since you are representing some yaml here, use yaml and github will use yaml highlighting.